src/components/Gouvernance/Membre/Membre.test.tsx

```diff
@@ -1,49 +1,495 @@
+      expect(plusDeDetails).not.toBeInTheDocument()
+    })
+
+    it.each(['', undefined])('=> alors un drawer s’ouvre et etant donner que le links.plusDetailsHref est %s alors il ne s’affiche pas', (value) => {
+      // GIVEN
+      afficherGouvernance({
+        membres: [
+          {
+            contactReferent: {
+              denomination: 'Contact politique de la collectivité',
+              mailContact: 'julien.deschamps@rhones.gouv.fr',
+              nom: 'Henrich',
+              poste: 'chargé de mission',
+              prenom: 'Laetitia',
+            },
+            feuillesDeRoute: [{ montantSubventionAccorde: 5_000, montantSubventionFormationAccorde: 5_000, nom: 'Feuille de route inclusion' }, { montantSubventionAccorde: 5_000, montantSubventionFormationAccorde: 5_000, nom: 'Feuille de route numérique du Rhône' }],
+            links: { plusDetails: value },
+            nom: 'Préfecture du Rhône',
+            roles: ['Co-porteur'],
+            telephone: '+33 4 45 00 45 00',
+            type: 'Administration',
+            typologieMembre: 'Préfecture départementale',
+          },
+        ],
+      })
+
+      // WHEN
+      jOuvreLesDetailsDuMembre('Préfecture du Rhône')
+      // THEN
+      const drawer = screen.getByRole('dialog', { name: 'Préfecture du Rhône' })
+      const plusDeDetails = within(drawer).queryByText(matchWithoutMarkup('Plus de détails'))
+      expect(plusDeDetails).not.toBeInTheDocument()
+    })
   })
 
-  function afficherGouvernance(): void {
-    const gouvernanceViewModel = gouvernancePresenter(gouvernanceReadModelFactory(), new Date())
-    render(<Gouvernance gouvernanceViewModel={gouvernanceViewModel} />)
-  }
+  describe('quand je clique sur le membre autre que la préfecture,', () => {
+    it('=> puis que je clique sur fermer, alors le drawer se ferme', () => {
+      // GIVEN
+      afficherGouvernance()
+
+      // WHEN
+      jOuvreLesDetailsDuMembre('Département du Rhône')
+      const drawer = screen.queryByRole('dialog', { name: 'Département du Rhône' })
+      jeFermeLesDetailsDuMembre('Fermer les détails du membre : Département du Rhône')
+
+      // THEN
+      expect(drawer).not.toBeVisible()
+    })
 
-  function jOuvreLesDetailsDUnMembre(): void {
-    fireEvent.click(screen.getByRole('button', { name: 'Préfecture du Rhône' }))
-  }
+    it('=> alors un drawer s’ouvre avec les détails du membre rempli totalement', () => {
+      // GIVEN
+      afficherGouvernance()
 
-  function jeFermeLesDetailsDUnMembre(): void {
-    fireEvent.click(screen.getByRole('button', { name: 'Fermer les détails du membre : Préfecture du Rhône' }))
-  }
+      // WHEN
+      jOuvreLesDetailsDuMembre('Département du Rhône')
+
+      // THEN
+      const drawer = screen.getByRole('dialog', { name: 'Département du Rhône' })
+      expect(drawer).toHaveAttribute('aria-labelledby', 'labelMembreId')
+      const totaltirets = within(drawer).queryByText('-')
```

Comment:
```suggestion
      const absenceDeDonnee = within(drawer).queryByText('-')
```

Autrement, c'est un détail d'implémentation.